opentracing_shim/src/main/java/io/opentelemetry/opentracingshim/SpanBuilderShim.java

```diff
@@ -210,9 +210,10 @@ public Span start() {
 
     SpanShim spanShim = new SpanShim(telemetryInfo(), span);
 
-    if (distContext != null && distContext != telemetryInfo().emptyDistributedContext()) {
-      spanContextTable().create(spanShim, distContext);
+    if (distContext == null) {
+      distContext = telemetryInfo().emptyDistributedContext();
```

Comment:
Here we try to NOT write to `spanContextTable()` as it's a global write lock (we do similar things for other cases where an empty distributed context is used, as it means it's basically a no-op object). 

Maybe you are seeing a specific scenario?